.gitmodules

```diff
@@ -7,3 +7,9 @@
 [submodule "lib/i18next"]
 	path = lib/i18next
 	url = https://github.com/i18next/i18next.git
+[submodule "lib/bootstrap"]
+	path = lib/bootstrap
+	url = https://github.com/twbs/bootstrap-sass/
+[submodule "lib/magnific-popup"]
+	path = lib/magnific-popup
+	url = https://github.com/dimsemenov/Magnific-Popup
```

Comment:
While using submodules in git is a working solution to dependency management, I don't think it's the right one. Technically speaking, these aren't submodules, as in separated parts of this project, but dependencies - we'd be better off using bower for dependency resolution. It was created for this sort of stuff, and many people use it.